spec/lib/efile/md/md502_r_calculator_spec.rb

```diff
@@ -3,7 +3,13 @@
 describe Efile::Md::Md502RCalculator do
   let(:filing_status) { "single" }
   # df_data_2_w2s has $8000 in federal social security benefits
-  let(:intake) { create(:state_file_md_intake, :df_data_2_w2s, filing_status: filing_status) }
+  let(:intake) {
+    if filing_status == 'married_filing_jointly'
+      create(:state_file_md_intake, :df_data_2_w2s, :with_spouse, filing_status: filing_status)
+    else
+      create(:state_file_md_intake, :df_data_2_w2s, filing_status: filing_status)
+    end
+  }
```

Comment:
this one did _not_ like the conditional setting in MD502SU spec, not sure why -- maybe need to try again 😓